### ISSUE TITLE
Music based editing

### ISSUE THREAD

**Issue by flofriday** [DevonCrawford/Video-Editing-Automation#7]
You feed the program a music clip along with the video files and it cuts the video synced with the music.

**Comment by Osnott** [comment]
Yes! Like, check for loud kicks in the audio file with a low frequency and cut along those times so it is synced up with the kicks.

**Comment by open-all** [comment]
To add to osnott we should slice the video feed based on transients. There could also be a function that allows you to set the threshold of said transients in audio clips.

**Comment by JoshuaDoes** [comment]
tfw I literally just got done suggesting this in the comments of the YouTube video before checking out the source
